### DISCUSSION TITLE
Webpack publicPath added to relative paths

### DISCUSSION THREAD

**Discussion by Leo-Felde** [quasarframework/quasar#17348] [CLI - PWA mode]
Hello, I'm trying to host my quasar webpack pwa in a subdirectory of my webserver `<protocol>://<domain>/motorista`

I added in my quasar.config.js `build.publicPath = "motorista"` which makes so the server can properly access the pages, manifest and service-worker, however, none of the assets like vendors.js and .css files are found because all the relative paths add the publicPath, for example:

/my-project/motorista/home/index.html has `<script defer="" src="/motorista/js/vendor.js">` which results in 404, it should not contain the public path because /motorista only contains the page's index.htmls and everything else in /my-project

Am I not configuring something or maybe doing something wrong? Any help is appreciated

**Comment by ivanjaros** [comment]
try in quasar config file, in the build section to use this:
```
publicPath: ctx.prod ? 'https://foo.com/motorista' : "/",
```

it should point to local assets on dev server and to the absolute path in production.
